EPAM_iOS_School_Practice_2_Star_Wars/EPAM_iOS_School_Practice_2_Star_Wars/RecentPersonsController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  RecentPersonsController.swift
+//  EPAM_iOS_School_Practice_2_Star_Wars
+//
+//  Created by Maxim on 14.02.2020.
+//  Copyright © 2020 Maksim Svyatov. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+
+class RecentPersonsController: UITableViewController {
+    
+    var personService: PersonServiceProtocol = PersonService()
+    var recentItems: Results<Person>!
+    var selectedItem: Person?
+    
+    @IBOutlet weak var recentsLabel: UILabel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        showRecentItems()
+        showRecentLabelText()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        showRecentLabelText()
+        showRecentItems()
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        guard let count = recentItems?.count else { return 0 }
```

Comment:
Best practice when `return` in new line. 
This approach is more convenient for debugging.

Reply:
Fixed